test/test_t8code_2d.jl

```diff
@@ -39,14 +39,17 @@ end
 end
 
 @trixi_testset "test check_for_negative_volumes" begin
-    @test_throws "Discovered negative volumes" begin
+    # test is currently broken as t8code applies a correction on the fly
+    # @test_throws "Discovered negative volumes" begin
+    @test begin
```

Comment:
It looks like this is not broken anymore, correct? Could you please fix the comment?

Reply:
@jmark @benegee You can do that in one of your new PRs working on T8code stuff in Trixi.jl

Reply:
The story behind this is a bit more involved. We will elaborate the comment a bit more.